integration/pom.xml

```diff
@@ -820,14 +820,17 @@
             </modules>
         </profile>
         <profile>
+            <!--
+               The profile with which build runs in WSO2 Jenkins
+                    -->
             <id>full</id>
             <modules>
                 <module>samples/product</module>
                 <module>automation-extensions</module>
                 <module>tests-common</module>
                 <module>mediation-tests</module>
                 <module>cli-tests</module>
-<!--                <module>dataservice-hosting-tests</module>-->
+                <module>dataservice-hosting-tests</module>
```

Comment:
Shall we enable dss tests in the smoke profile aswell?

Reply:
They are not fixed yet and travis will fail for the PRs